src/V3DfgDfgToAst.cpp

```diff
@@ -100,21 +100,25 @@ AstShiftRS* makeNode<AstShiftRS, DfgShiftRS, AstNodeExpr*, AstNodeExpr*>(  //
 template <>
 AstCCast* makeNode<AstCCast, DfgCCast, AstNodeExpr*>(const DfgCCast* vtxp, AstNodeExpr*) {
     vtxp->v3fatalSrc("not implemented");
+    return nullptr;
```

Comment:
Please put fixing these into a "fix MSVC" first pull.

I think instead we should have the unreachable marking that v3fatalSrc uses be recognized by MSVC see https://en.cppreference.com/w/cpp/utility/unreachable.  We can't use the C++22 feature yet, so use __assume(false) as described there into VL_UNREACHABLE.

If the return is STILL needed after that is fixed then use

    return nullptr;  // LCOV_EXCL_LINE